docs/sources/api-response-examples/atlassian/jira/sanitized/issue_v3.json

```diff
@@ -0,0 +1,182 @@
+    "customfield_10703": null,
+    "customfield_10704": null,
+    "resolutiondate": null,
+    "customfield_10705": null,
+    "customfield_10706": null,
+    "customfield_10707": null,
+    "workratio": -1,
+    "customfield_10708": null,
+    "customfield_10709": null,
+    "issuerestriction": {
+      "issuerestrictions": {},
+      "shouldDisplay": false
+    },
+    "lastViewed": "2023-06-14T06:18:52.375-0400",
+    "watches": {
+      "watchCount": 1,
+      "isWatching": true
+    },
+    "created": "2023-06-09T06:37:10.762-0400",
+    "priority": {
+      "id": "3"
+    },
+    "customfield_10100": null,
+    "labels": [],
+    "timeestimate": 489600,
+    "aggregatetimeoriginalestimate": null,
+    "versions": [],
+    "issuelinks": [],
+    "assignee": null,
+    "updated": "2023-06-14T06:19:04.368-0400",
+    "status": {
+      "id": "10000",
+      "statusCategory": {
+        "id": 2,
+        "key": "new",
+        "colorName": "blue-gray"
+      }
+    },
+    "components": [],
+    "timeoriginalestimate": null,
+    "timetracking": {
+      "remainingEstimate": "3w 2d",
+      "timeSpent": "1d",
+      "remainingEstimateSeconds": 489600,
+      "timeSpentSeconds": 28800
+    },
+    "customfield_10720": null,
+    "customfield_10600": null,
+    "security": null,
+    "customfield_10008": null,
+    "aggregatetimeestimate": 489600,
+    "attachment": [
+      {
+        "id": "10000",
+        "filename": "03 - getting into action.mp3",
+        "author": {
+          "accountId": "{\"hash\":\"ZSiF2ksLmucJypprVlOuvn9YZPtcViktBHNWFxvBjco\"}",
+          "emailAddress": "{\"scope\":\"email\",\"domain\":\"worklytics.co\",\"hash\":\"GItqpD0PrNe1WX8MpyKYmCPjh99MGEBG30ziv65GIuw\"}",
+          "active": true,
+          "timeZone": "America/New_York",
+          "accountType": "atlassian"
+        },
+        "created": "2023-06-09T12:31:44.105-0400",
+        "size": 2829492,
+        "mimeType": "audio/mpeg",
+        "content": "https://api.atlassian.com/ex/jira/f6eef702-e05d-43ba-bd5c-75fce47d560e/rest/api/3/attachment/content/10000"
+      }
+    ],
+    "summary": "worklog test",
+    "creator": {
+      "accountId": "{\"hash\":\"ZSiF2ksLmucJypprVlOuvn9YZPtcViktBHNWFxvBjco\"}",
+      "emailAddress": "{\"scope\":\"email\",\"domain\":\"worklytics.co\",\"hash\":\"GItqpD0PrNe1WX8MpyKYmCPjh99MGEBG30ziv65GIuw\"}",
+      "active": true,
+      "timeZone": "America/New_York",
+      "accountType": "atlassian"
+    },
+    "subtasks": [],
+    "reporter": {
+      "accountId": "{\"hash\":\"ZSiF2ksLmucJypprVlOuvn9YZPtcViktBHNWFxvBjco\"}",
+      "emailAddress": "{\"scope\":\"email\",\"domain\":\"worklytics.co\",\"hash\":\"GItqpD0PrNe1WX8MpyKYmCPjh99MGEBG30ziv65GIuw\"}",
+      "active": true,
+      "timeZone": "America/New_York",
+      "accountType": "atlassian"
+    },
+    "aggregateprogress": {
+      "progress": 28800,
+      "total": 518400,
+      "percent": 5
+    },
+    "customfield_10000": null,
```

Comment:
plaintext values of any custom fields likely need to be redacted by default, just to be safe; numbers likely OK, but otherwise risk/reward not worth it until we're using this

Reply:
For these endpoints (retrieving a single issue from cloud/server) I've added response metadata, so we do not need to care about custom fields -which we can't know how they will be-